src/SFA.DAS.AANHub.Data/Repositories/ApprenticesReadRepository.cs

```diff
@@ -17,5 +17,10 @@ internal class ApprenticesReadRepository : IApprenticesReadRepository
                 .AsNoTracking().Where(a => a.ApprenticeId == apprenticeId)
                 .SingleOrDefaultAsync();
 
+        public async Task<Apprentice?> GetApprenticeByApprenticeIdAndUserId(long apprenticeId, Guid? userId) => await _aanDataContext
```

Comment:
`userId` should probably be `memberId` to match the property.

Reply:
Really sorry for this. `GetApprenticeByApprenticeIdAndUserId` is no longer applicable as discussed with Deven on an earlier huddle.  MemberId is the member applying the PATCH and apprenticeId is the object getting patched. The function is not getting called anywhere and should not have gone in.